src/style.css

```diff
@@ -0,0 +1,173 @@
+.body {
+  margin: 0;
+  padding: 0;
+}
+
+.product__card {
+  position: relative;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  position: absolute;
+  width: 160px;
+  height: 134px;
+  left: calc(50% - 160px / 2);
+  top: 33px;
+}
+
+.card__name {
+  position: absolute;
```

Comment:
remove positioning from all elements, use margins and let them stay in document flow